### PR TITLE
Add: New ML Learning resource

### LINKED ISSUE #2039: [Add]: New ML Learning Course Playlist
### Site URL

https://www.youtube.com/@IndianAIProduction

### Description of the above provided site

This is one of the greatest Machine Learning learning course for beginners to start learning Machine Learning from scratch.

### Category

Youtube

### Sub Category

Machine Learning

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Bratajit-03**
## Fixes Issue
Closes #2039 

Added a new Machine Learning Resource from Youtube

### CODE REVIEW

File: database/youtube/machine-learning.json [hunk 1]
```diff
@@ -94,5 +94,13 @@
     "category" : "youtube",
     "subcategory": "machine-learning",
     "language": "english"
+  },
+  {
+    "name" : "Indian AI Production",
+    "description": "This channel provides a complete guide to Machine Learning from Beginner to Advanced and clearing all latest concepts and technologies related to Data Science.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:100]:
```suggestion
    "description": "This channel provides a complete guide to Machine Learning from beginner to advanced and teaches the latest concepts and technologies related to Data Science.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Bratajit-03! :) I gave some feedback on your description. Other than that you're off to a great start

**Comment by Bratajit-03** [comment]
@CBID2 sir, i have done the necessary changes. please give it a view and allow it to merge.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Anmol-Baranwal** [comment]
> Well done @Bratajit-03! Note: I go by she/her 😃

LOL! 🤣

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**